birdhouse/components/proxy/conf.d/cors.include.template

```diff
@@ -0,0 +1,42 @@
+    proxy_hide_header 'Access-Control-Allow-Origin';
+
+    set $access_control_allow_origin "${BIRDHOUSE_PROXY_CORS_ALLOW_ORIGIN}";
+    if ( $access_control_allow_origin ~ "^$" ) {
+       set $access_control_allow_origin '*';
+    }
+
+    set $vary_origin ""; # nginx will omit this header value if set to the empty string
+    if ( $access_control_allow_origin != '*' ) {
+       set $vary_origin Origin;
+    }
```

Comment:
If problem with the if-statement to set default value, does this `$vary_origin` also generate uninitialised warning too?